mlflow/server/handlers.py

```diff
@@ -773,7 +1125,18 @@ def _log_batch():
 @catch_mlflow_exception
 @_disable_if_artifacts_only
 def _log_model():
-    request_message = _get_request_message(LogModel())
+    request_message = _get_request_message(
+        LogModel(),
+        schema = {
+            "run_id": [
+                _assert_string,
+                _assert_required
+            ],
+            "model_json": [
+                _assert_string
```

Comment:
While it's not documented, `model_json` is required too. Can we add `_assert_required`?